### PR TITLE
Eliminate dead LogicalOperators

### PR DESCRIPTION
**Pull request by nicks**
Hi! I hope you'll accept this feature-request-as-pull-request which simplifies `&&` and `||` operators.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -102,6 +102,27 @@ export default function () {
       }
     },
 
+    LogicalExpression: {
+      exit(path) {
+        const { node } = path;
+        var operator = path.get("operator").node;
+        var leftOfOperatorTest = path.get("left").evaluateTruthy();
+        if (operator == "&&") {
```
Comment (erikdesjardins) [inline src/index.js]: Nit: I prefer `===` for this and the other `operator == "||"` test below

**Comment by erikdesjardins** [comment]
LGTM modulo one nit

File: src/index.js (hunk 1 above)
Comment (nicks) [inline src/index.js]: fixed!

**Comment by erikdesjardins** [comment]
Thanks!

I also just tested this on the https://github.com/honestbleeps/Reddit-Enhancement-Suite codebase and a React/Relay SPA (nothing broke), so look forward to a 2.1.0 release in the next hour.

**Pull request merged**